.github/workflows/ci.yml

```
@@ -12,39 +15,38 @@ jobs:
-          git add .
-          git commit -m "Apply pre-commit fixes" || echo "No changes to commit"
-
-      - name: Build the project
-        run: pnpm build
+      - run: pnpm prettier:check
+      - run: pnpm build
+  test-enterprise:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)